index.css

```diff
@@ -108,4 +108,10 @@
 
 .monospaced {
   font-feature-settings: "tnum" on;
+}
+
+.overflow-ellipsis {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
```

Comment:
Don't we have any linting rules on this repository? 😅

Reply:
No, in this repository we don't have linting at the moment 😅